webapp/packages/plugin-data-viewer/src/DatabaseDataModel/Actions/ResultSet/compareResultSetRowKeys.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import type { IResultSetRowKey } from './IResultSetDataKey';
+
+export function compareResultSetRowKeys(a: IResultSetRowKey, b: IResultSetRowKey): number {
+  return a.index + a.subIndex / 10 - b.index - b.subIndex / 10;
```

Comment:
Probably we need some comment here or create two named variables to it will be easier to read

Reply:
comment is added